setup.py

```diff
@@ -33,5 +33,6 @@
     extras_require={
         # We can add the mesos specific dependencies here
         'mesos_executor': ['pymesos'],
+        'metrics': ['yelp-meteorite', 'mock']
```

Comment:
let's avoid a runtime dependency on mock if we can. We can emulate whichever portion of the yelp-meteorite API we use :-)